src/dialogs/more-info/ha-more-info-history.ts

```diff
@@ -59,6 +60,10 @@ export class MoreInfoHistory extends LitElement {
     if (changedProps.has("entityId")) {
       this._stateHistory = undefined;
 
+      this._showMoreHref = `/history?entity_id=${
+        this.entityId
+      }&start_date=${startOfYesterday().toISOString()}`;
```

Comment:
Move it below the `if (!this.entityId)` check

Reply:
But that would mean that if we generated a HREF and then the entity ID is getting undefined, the HREF would still contain the old entity ID resulting in a wrong link? We could completely clear the HREF in that case, so no navigation happens anymore?

Reply:
We dont render when the entity id is undefined